app/controllers/api/v1/reviews_controller.rb

```diff
@@ -1,26 +1,51 @@
 class Api::V1::ReviewsController < ApiController
   protect_from_forgery unless: -> {request.format.json? }
-  before_action :authenticate_user!, except: [:show]
-
-  def show 
-    render json: review.all
-  end
-
+  before_action :authenticate_user
+  before_action :authorize_user, except: [:create]
+  
   def create
     review = Review.new(review_params)
     review.sauce_id = params[:sauce_id]
     review.user_id = current_user[:id]
 
     if review.save
-      render json: {review: review}
+      votes = review.votes
+      payload = {**review.attributes, votes: votes}
+      render json: {review: payload}
     else 
       render json: {error: review.errors.full_messages, status: :unprocessable_entity }
     end
   end
 
+  def destroy
+    review = Review.find(params[:id])
+
+    destroy_review(review)
```

Comment:
this `destroy` action is small enough that we can just have the contents of `destroy_review` right in here.

Reply:
moved logic back into "destroy" method and deleted "destroy_review"